components/dashboard/src/App.tsx

```diff
@@ -53,6 +53,15 @@ function isGitpodIo() {
     return window.location.hostname === 'gitpod.io' || window.location.hostname === 'gitpod-staging.com' || window.location.hostname.endsWith('gitpod-dev.com') || window.location.hostname.endsWith('gitpod-io-dev.com')
 }
 
+function isWebsiteSlug(pathName: string) {
+    const slugs = ['chat', 'features', 'screencasts', 'blog', 'docs', 'changelog', 'pricing', 'self-hosted', 'gitpod-vs-github-codespaces', 'support', 'about', 'careers', 'contact', 'media-kit', 'imprint', 'terms', 'privacy']
+    return slugs.some(slug => pathName.startsWith('/' + slug + '/'));
```

Comment:
**issue**: Should we also redirect URLs without a trailing `/`? For example, [`/changelog/`](https://se-website-redirect.staging.gitpod-dev.com/changelog/) v. [`/changelog`](https://se-website-redirect.staging.gitpod-dev.com/changelog).

Reply:
**change:** add /values as well please 🧡

[Copying/pasting the link](https://news.ycombinator.com/item?id=28432300) from notion is getting old.